README.md

```diff
@@ -37,27 +37,24 @@ There are multiple ways to install PSReadLine.
 
 ### Install from PowerShellGallery (preferred)
 
-You will need PowerShellGet.  It is included in Windows 10 and [WMF5](http://go.microsoft.com/fwlink/?LinkId=398175). If you are using PowerShell V3 or V4, you will need to install [PowerShellGet](https://www.microsoft.com/en-us/download/details.aspx?id=49186).
+You will need PowerShellGet.  It is included in Windows 10 and [WMF5](http://go.microsoft.com/fwlink/?LinkId=398175). If you are using PowerShell V3 or V4, you will need to install [PowerShellGet](https://docs.microsoft.com/powershell/gallery/installing-psget).
 
 After installing PowerShellGet, you can simply run `Install-Module PSReadLine`.
 
 If you are on Windows 10, PSReadLine is already installed. Windows 10 RTM and the November update have version 1.1, later builds have version 1.2 (which includes vi mode). See below for how to upgrade.
 
 ### Install from GitHub (deprecated)
 
-With the preview release of PowerShellGet for PowerShell V3/V4, I am deprecating downloads from GitHub.  I don't intend to update releases on GitHub, and may remove the release entirely from GitHub at some point.
-
-To install the build from GitHub, you can install using [PsGet](http://psget.net) (very easy to install), and run `Install-Module PSReadLine`.  Note that PsGet and PowerShellGet both have a Install-Module command, but they are very different despite having similar names and commands. PowerShellGet is implemented and supported by the PowerShell team at Microsoft, PsGet is a 3rd party tool that was a source of inspiration for PowerShellGet.
-
-If you'd rather not use PsGet, you can just download the file [PSReadLine.zip](https://github.com/lzybkr/PSReadLine/releases/download/Latest/PSReadLine.zip) and extract the contents into your `C:\Users\[User]\Documents\WindowsPowerShell\modules\PSReadLine` folder. (You may have to create these directories if they don't exist.)
+With the preview release of PowerShellGet for PowerShell V3/V4, I am deprecating downloads from GitHub.
```

Comment:
Should look for occurrences of `I` and change to something not individual specific.  LIke:

`...downloads from GitHub are deprecated.`

Reply:
Sure thing. Will do.

Reply:
Other `I` instances are in the code comments, and I think we should keep them as they are.
For the CI badge URL, I will change them when moving the repo to PowerShell org. Otherwise, CI badges won't work.